pkg/meta/meta.go

```diff
@@ -1022,6 +1022,49 @@ func (m *Meta) ListTables(dbID int64) ([]*model.TableInfo, error) {
 	return tables, nil
 }
 
+// ListTablesByFilter return all tables that does not filtered by filter in a database.
```

Comment:
please add a comment to note that this method is used by the ticdc, to prevent deleted